src/Uno.UI/Microsoft/UI/Xaml/Controls/ProgressRing/ProgressRing.xaml

```diff
@@ -15,6 +15,7 @@
 		<Setter Property="MinWidth" Value="16" />
 		<Setter Property="IsTabStop" Value="False" />
 		<Setter Property="Width" Value="32" />
+        <Setter Property="Height" Value="32" />
```

Comment:
@MartinZikmund do we have this to change in some other places (fluent styles maybe)?

Reply:
I think so, although the code there differs more than I expected (e.g. the `Width` setter is not there either)

Reply:
@MartinZikmund I can't see what to change in fluent styles.

I assume that we should be okay as long as it works with "FL" checked in SamplesApp?

![image](https://user-images.githubusercontent.com/31348972/225544794-ca5989e1-bec7-4cc9-a22b-24ebb89a6959.png)

Reply:
On this same sample page you could previously repro the stretching (without Fluent checked)? If yes, then I guess the issue does not show up on Fluent styles 🤔

Reply:
Before the change, I was able to reproduce both with FL checked/unchecked.
After the change, it works on both cases.

Reply:
This would mean that this change uses style merging, as this style:
https://github.com/unoplatform/uno/blob/master/src/Uno.UI.FluentTheme.v2/Resources/Version2/PriorityDefault/ProgressRing.xaml#L9

Does not define any size, but the original file is quite different (and defines min values as well):
https://github.dev/microsoft/microsoft-ui-xaml/blob/8390ab44f93e643434059d45aefb29b2f7070a87/dev/ProgressRing/ProgressRing_v1.xaml#L8-L19

It could be that we're behind on the fluent style for ProgressRing.

Reply:
@jeromelaban ProgressRing in general is either behind or completely different than WinUI. But I think as long as this currently works and fixes the vertical stretching, it should be good to merge?

Reply:
It fixes the issue at this point, so let's merge as is.